src/utils/mixpanel.ts

```diff
@@ -0,0 +1,19 @@
+import { MIXPANEL_PROJECT_TOKEN } from "secrets";
+
+export const initMixpanel = (opts?: Object) => {
+  if (!MIXPANEL_PROJECT_TOKEN) return;
```

Comment:
can we log a warning to the console about the mixpanel not being set up correctly?
i guess it's enough to have this log in the init function only.

Reply:
fixed in b54905562aeb508beb223e2d87710f875297d159